modelzoo/DeepFM/train.py

```diff
@@ -579,7 +819,7 @@ def main(tf_config=None, server=None):
                     'index': task_index,
                     'is_chief': is_chief
                 },
-                     server=server)
+                    server=server)
```

Comment:
Please use `generate_cluster_info()` to Parse distributed training configuration and generate cluster information